CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
```

Comment:
This is more a feature than a bugfix, since the exception is new.

Reply:
Done, thanks!